docs/GettingStarted.rst

```diff
@@ -1361,6 +1363,18 @@ You can look for articles easily using the available search bar in the topbar of
 
 |image191|
 
+
+News application views
+^^^^^^^^^^^^^^^^^^^^^^^
+
+Different views are available from News application so you can easily find pinned articles, your posted articles, your scheduled articles, your drafts or your archived articles.
```

Comment:
You anticipate, right ? :)

Reply:
Yes. To have the intro for the next documentation PRs.